gnuradio-runtime/include/gnuradio/block.h

```diff
@@ -177,7 +177,7 @@ namespace gr {
      *
      * WORK_CALLED_PRODUCE is used where not all outputs produce the
      * same number of items. general_work must call produce() for each
-     * output to indicate the numer of items actually produced.
+     * output to indicate the number of items actually produced.
```

Comment:
Should this be `number` or `numer` ? I ask because there are other places in the source code that have `numer`

Reply:
number